sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -1041,6 +1042,23 @@ def test_output_typehints(self):
               ShardedKeyType[typehints.Tuple[int, int]],  # type: ignore[misc]
               typehints.Iterable[str]])
 
+  def test_with_type_hints(self):
+    options = PipelineOptions()
+    options.view_as(TypeOptions).runtime_type_check = True
+    with TestPipeline(options=options) as pipeline:
+      collection = pipeline \
```

Comment:
nit / personal opinion: you can use brackets instead of line continuation tokens

Reply:
Love it, done.

YAPF seems to be way behind `black` in this regard, I *think* I remember seeing somewhere you folks talking about migrating to black, was it a dead end?

Reply:
We're still on yapf as far as our style infra, idk if anyone is actively investigating alternatives right now